assets/css/_leaflet.scss

```diff
@@ -7,9 +7,14 @@ $max-leaflet-z-index: 1000;
 }
 
 .c-vehicle-map-state--street-view-enabled + .leaflet-container {
-  cursor: url("data:image/svg+xml,%3Csvg height='114' width='114' xmlns='http://www.w3.org/2000/svg'%3E%3Ccircle cx='57' cy='57' r='57' fill='%230000FF' opacity='0.25'/%3E%3Ccircle cx='57' cy='57' r='5' fill='%235182FF' stroke='%230016DB' stroke-width='2' /%3E%3C/svg%3E%0A")
-      57 57,
+  cursor: url("data:image/svg+xml,%3Csvg width='20' height='20' viewBox='0 0 20 20' fill='none' xmlns='http://www.w3.org/2000/svg'%3E%3Cg id='street-view-cursor'%3E%3Ccircle id='Ellipse 5' cx='10' cy='10' r='10' fill='%236229A3'/%3E%3Cpath id='Vector' d='M11.3488 5.62462C12.094 5.62462 12.6986 5.03681 12.6986 4.31231C12.6986 3.5878 12.094 3 11.3488 3C10.6036 3 9.99903 3.5878 9.99903 4.31231C9.99903 5.03681 10.6036 5.62462 11.3488 5.62462ZM14.0062 9.70097L13.351 9.37836L13.0782 8.57457C12.6648 7.35522 11.5119 6.50222 10.2043 6.49949C9.19197 6.49675 8.63238 6.77562 7.58068 7.18845C6.97329 7.4263 6.47556 7.87741 6.18311 8.45155L5.9947 8.82337C5.77536 9.25533 5.95252 9.78026 6.39401 9.99624C6.83269 10.2122 7.36697 10.0373 7.58912 9.60528L7.77753 9.23346C7.87595 9.04208 8.03904 8.89172 8.24151 8.81243L8.99513 8.51716L8.5677 10.1767C8.42148 10.7453 8.57895 11.3496 8.9867 11.7843L10.6711 13.5723C10.8736 13.7883 11.017 14.048 11.0901 14.3296L11.6047 16.3336C11.7256 16.8011 12.2149 17.0882 12.6958 16.9706C13.1766 16.853 13.4719 16.3773 13.351 15.9098L12.7267 13.4766C12.6536 13.195 12.5102 12.9325 12.3077 12.7193L11.0282 11.3605L11.5119 9.48225L11.6666 9.93336C11.8156 10.3735 12.1362 10.7371 12.558 10.9449L13.2132 11.2675C13.6518 11.4835 14.1861 11.3085 14.4083 10.8766C14.6248 10.4473 14.4476 9.91696 14.0062 9.70097ZM7.56944 13.5477C7.47945 13.7691 7.34447 13.9687 7.17013 14.1355L5.76412 15.5052C5.41261 15.8469 5.41261 16.4019 5.76412 16.7437C6.11562 17.0854 6.68365 17.0854 7.03515 16.7437L8.70549 15.1197C8.87703 14.9529 9.012 14.7534 9.1048 14.5319L9.48443 13.6078C7.92938 11.9592 8.39617 12.465 8.15153 12.1397L7.56944 13.5477Z' fill='white'/%3E%3C/g%3E%3C/svg%3E")
```

Comment:
I'd _really_ like to **not** use a data url for this SVG, but I couldn't figure out how to get our webpack sass loader to comply with relative url's.

Reply:
Yeah, I ran into this when implementing the original as well and didn't have time to dig into it too deeply.